index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>  
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>SMART STARTER APP</title>    
+  </head>
+  <body style="margin: none;">
+    <h2>SMART STARTER APP ( PATIENT DEMOGRAPHICS )</h2>
+    <div id='errors'>
+    </div>
+    <div id='holder'>
+    </div>
+   
+  </body>
+  <script src="./lib/fhir-client.js"></script>

```

Comment:
nit: can we use the minified version of the fhir-client to be consistent with the jquery.min.js lib file?